lib/srv/reexec.go

```diff
@@ -189,6 +195,10 @@ func RunCommand() (errw io.Writer, code int, err error) {
 	if contfd == nil {
 		return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("continue pipe not found")
 	}
+	termiantefd := os.NewFile(TerminateFile, "/proc/self/fd/8")
```

Comment:
Why `fd/8`?

Reply:
Because it was free, and Linux didn't complain about it. It should be `5` 🙈 
I changed the code to use the same constants.